pkg/skaffold/docker/client.go

```diff
@@ -43,10 +46,15 @@ var (
 	dockerAPIClientOnce sync.Once
 	dockerAPIClient     APIClient
 	dockerAPIClientErr  error
+	procVerDetect       bool
+	isWsl               bool
```

Comment:
I'm not a fan of having this at both the package level and set through a function. Could we drop this here and always return the result of detectWsl?

Reply:
then we don't cache. Ok.